modules/test/playwright/liferay.config.ts

```diff
@@ -11,7 +11,7 @@ const liferayConfig = {
 	},
 	user: {
 		login: process.env.LIFERAY_USER_LOGIN || 'test@liferay.com',
-		password: process.env.LIFERAY_USER_PASSWORD || 'test',
+		password: process.env.LIFERAY_USER_PASSWORD || 'test1',
```

Comment:
This is interesting, bc I'm preparing a PR for this kind of things.

In any case, why did you chose to change the password here? Because you use test1 in your computer? :thinking: 

I guess that's not going in the final PR, right?

Reply:
problem is how to ensure user/password exists in the target system. Probably, depending on the configuration, system forces you to change password upon first login. So, if we want to reuse default test password (which seems sensible to me at this point) we might ensure a config that does not force user to do anything after login.

Reply:
In my PR I assume the fixture takes care of this by either creating the user itself or bc it comes prepopulated in the DB.

However, as the code is now, it is taken from the environment, so there's no need to do any change here, just set the proper environment when the test is run.

In general I don't think the tests can account for any state of the server. We should assume controlled initial states for the tests to run. Otherwise there will always be possibilities for failure :shrug:

Reply:
It's a typo, sorry, I'll fix it. 

I totally agree with the assumption for tests to run in an fresh, new build (as it gonna happen in the pipelines, I guess)

Creating a new user before running tests sounds great, but we should return to a fresh database while working on development in a daily basis. What do you think?

Reply:
Normally, what matters is the roles granted to the user are enough to exercise the features we're testing. For the DSM, we've only worried about administrator role, and that's enough for the moment. In the future, with permissions around the corner, we'll have to fine tune this, but let's work on one thing at a time